test/module/irohad/ametsuchi/ametsuchi_fixture.hpp

```diff
@@ -50,13 +51,42 @@ namespace iroha {
             std::make_unique<InMemoryBlockStorageFactory>();
         auto reconnection_strategy_factory = std::make_unique<
```

Comment:
Should this variable be made fixture static field? Otherwise its lifetime is not managed.